fastlane/Fastfile

```diff
@@ -386,6 +387,22 @@ platform :ios do
     )
   end
 
+  desc "Run LoadShedder tests"
+  lane :v3_loadshedder_integration_tests do |options|
+    replace_api_key_integration_tests
+    scan(
+      project: "./Tests/v3LoadShedderIntegration/v3LoadShedderIntegration.xcodeproj",
+      scheme: "v3LoadShedderIntegration",
+      derived_data_path: "scan_derived_data",
+      output_types: 'junit',
+      output_style: 'raw',
+      result_bundle: true,
+      configuration: 'Debug',
```

Comment:
I think running them on `Debug` is enough for now, since we are mostly just testing the interaction with the load shedder, which should be the same 👍

Reply:
+1. I'm not sure you even _could_ run them on release since we're using `@testable import`, but I haven't really checked. Won't make much of a difference in this case anyway.